requirements.txt

```diff
@@ -2,4 +2,6 @@ click==7.1.2
 modelmeta==0.2.0
 psycopg2==2.8.6
 SQLAlchemy==1.3.23
-xarray==0.16.2
\ No newline at end of file
+xarray==0.16.2
+toolz==0.11.1
+dask==2021.2.0
```

Comment:
If `toolz` is coming from `dask` I think you can use something like `dask[complete]` to ensure all the dependencies are installed.